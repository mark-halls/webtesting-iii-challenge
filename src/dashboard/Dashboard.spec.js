// Test away
import React from "react";
import { render } from "@testing-library/react";

import Dashboard from "./Dashboard";

test("it renders Dashboard without errors", () => {
  const { getByText } = render(<Dashboard />);
  expect(getByText(/locked/i)).toMatchSnapshot();
  expect(getByText(/open/i)).toMatchSnapshot();
});
