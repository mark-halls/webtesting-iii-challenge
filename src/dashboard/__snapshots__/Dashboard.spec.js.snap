// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders Dashboard without errors 1`] = `
<div
  class="led green-led"
>
  Unlocked
</div>
`;

exports[`it renders Dashboard without errors 2`] = `
<div
  class="led green-led"
>
  Open
</div>
`;
